frontend/src/app/shared/scss/_colors.scss

```diff
@@ -1,32 +1,62 @@
-/* Variables */
-$main-color: #6a983c;
-$main-dark-color: #46760a;
+/* Primary Color */
+$primary-color: #151515;
+$primary-color-010: #575757;
+$primary-color-020: #a9a9a9;
+$primary-color-030: #d1d1d1;
+$primary-color-040: #ebebeb;
+$primary-color-050: #f5f5f5;
+$primary-color-060: #f9f9f9;
+$primary-color-070: #ffffff;
```

Comment:
`$primary-color` l-as fi pus sa fie `verdele` (`$secondary-color `pe care l-ai facut mai jos)

Tu le-ai facut foarte frumos, doar ca o sa folosim foarte des culoarea principala.

Iar negru asta, in loc de `secondary-color`, l-as numi `grey-color`;

Reply:
fixed